Group15_FitnessAssignment/config/express.js

```diff
@@ -1,10 +1,11 @@
 // Module dependencies
 
 const express = require('express');
-//const session = require('express-session');
+const session = require('express-session');
```

Comment:
This dependency is new. You will need to run npm install before you can do nodemon. It adds session logic, which is needed for passport.